index.bs

```diff
@@ -3233,17 +3214,16 @@ This attestation statement format is used with FIDO U2F authenticators using the
     the attestation public key as |x5c|.
 
 : Verification procedure
-:: Given the [=verification procedure inputs=] |attStmt|, |authenticatorData| and |clientDataHash|, the verification procedure is
-    as follows:
+:: Given the [=verification procedure inputs=] |attStmt|, |authenticatorData| and
+    [=hash of the serialized client data=], the verification procedure is as follows:
```

Comment:
I don't think this needs to change, does it? On the other hand, the `|tbsHash|` intermediate step is now unnecessary. I therefore suggest reverting this change, eliminating step 4, and changing `|tbsHash|` to `|clientDataHash|` in step 6. You're welcome to ignore this if you disagree.